lib/x11utils.pm

```diff
@@ -86,6 +88,12 @@ sub ensure_unlocked_desktop {
                 select_user_gnome($username);
             }
         }
+        if (match_has_tag('guest-disabled-display')) {
+            wait_screen_change(sub {
+                    send_key 'shift';
+            }, 10);
+            record_info('Guest disabled display', 'Might be consequence of bsc#1168979');
```

Comment:
Since bugzilla is not available, I cannot see the content of bsc#1168992.
I suppose that you are addressing bsc#1168992 and that is different from bsc#1168979

This is wrong, it is not detecting bsc#1168979 unequivocally.


```suggestion
            record_info('Guest disabled display', 'Might be consequence of bsc#1168992');
```

Reply:
@SergioAtSUSE the record info is a weak reference to the bug, not stating that it would *detect* it. I am ok with this code as is.

Reply:
This is not https://bugzilla.opensuse.org/show_bug.cgi?id=1168992 as pressing `esc` or `shift` let us quit the "Guest disabled display" state.

Reply:
@ggardet thanks for confirming

Reply:
> @SergioAtSUSE the record info is a weak reference to the bug, not stating that it would _detect_ it. I am ok with this code as is.

This will record a soft-fail anytime ensure_unlocked_desktop finds a turned off display, which is not always a consequence of bsc#1168979, How can you ok with it when it can be done right?

Reply:
> > @SergioAtSUSE the record info is a weak reference to the bug, not stating that it would _detect_ it. I am ok with this code as is.
> 
> This will record a soft-fail anytime ensure_unlocked_desktop finds a turned off display, which is not always a consequence of bsc#1168979, How can you be ok with it when it can be done right?

Reply:
only for aarch64, and again, it's not stating that it is the actual reason, but for aarch64, for the time being, we _know_ that said bug, it's the cause for the behavior.

Reply:
> only for aarch64, and again, it's not stating that it is the actual reason, but for aarch64, for the time being, we _know_ that said bug, it's the cause for the behavior.

Let me use an example of why this approach is wrong.

1. We _know_ that one sympton of corona virus is fiber.
2. Let's diagnose every people with fiber as infected by coronavirus.

Reply:
> > only for aarch64, and again, it's not stating that it is the actual reason, but for aarch64, for the time being, we _know_ that said bug, it's the cause for the behavior.
> 
> Let me use an example of why this approach is wrong.
> 
>     1. We _know_ that one sympton of corona virus is fiber.
> 
>     2. Let's diagnose every people with fiber as infected by coronavirus.

More like 'It might be' corona :D so let's be careful